src/cclib/io/__init__.py

```diff
@@ -10,6 +10,7 @@
 from .cjsonwriter import CJSON
```

Comment:
Since this is related to the CJSON Writer, I have updated the CJSON Writer test suite pull request with this change. PR: https://github.com/cclib/cclib/pull/326